### PR TITLE
Null handling unification

### PR DESCRIPTION
**Pull request by lnkuiper**
Some more NULL handling unification, and verification code for default NULL handling for scalar and aggregate functions

### CODE REVIEW

File: src/execution/operator/aggregate/physical_simple_aggregate.cpp [hunk 1]
```diff
@@ -171,6 +177,8 @@ void PhysicalSimpleAggregate::Combine(ExecutionContext &context, GlobalSinkState
 
 		AggregateInputData aggr_input_data(aggregate.bind_info.get());
 		aggregate.function.combine(source_state, dest_state, aggr_input_data, 1);
+
+		gstate.state.counts[aggr_idx] += source.state.counts[aggr_idx];
```
Comment (Mytherin) [inline src/execution/operator/aggregate/physical_simple_aggregate.cpp:181]: Perhaps `#ifdef DEBUG` this line?

**Review by Mytherin — COMMENTED** [review]
Thanks for pushing this forward! LGTM, one minor comment:

File: src/execution/operator/aggregate/physical_simple_aggregate.cpp (hunk 1 above)
Comment (lnkuiper) [reply src/execution/operator/aggregate/physical_simple_aggregate.cpp:186]: Good idea. I `#ifdef DEBUG`'d all lines that pertain to the verification

**Comment by Mytherin** [comment]
Thanks!

**Pull request merged**